server.py

```diff
@@ -345,6 +345,13 @@ def main():
         quit(1)
     port = sys.argv[1]
 
+    databasePassword = input("Input database password: ")
```

Comment:
I assume by password you mean the key to decrypt the database?

Reply:
Yes, rn it is just a text string - so it can be `password123!` or a 256 char random string if you wanted.